packages/toolpad-studio/src/canvas/index.tsx

```diff
@@ -152,6 +151,15 @@ export default function AppCanvas({ basename, state: initialState }: AppCanvasPr
       return { nodes };
     });
 
+    setCommandHandler(bridge.canvasCommands, 'scrollComponent', (nodeId) => {
+      if (!nodeId) {
```

Comment:
Just another thing I noticed, not very important either: I noticed that when you call `scrollComponent` from the `RenderOverlay` component you already check if `selectedNode` is defined. So I guess that we don't need to check if `nodeId` is set here?